DiscoveryServiceSpecification/V3.0_SSP-001.yaml

```diff
@@ -11,83 +11,62 @@ info:
   license: 
     name: CC BY 4.0
     url: https://creativecommons.org/licenses/by/4.0/
-  version: V3.1.0_SSP-001
-  x-profile-identifier: https://admin-shell.io/aas/API/3/1/DiscoveryServiceSpecification/SSP-001
+  version: V3.0.1_SSP-001
+  x-profile-identifier: https://admin-shell.io/aas/API/3/0/DiscoveryServiceSpecification/SSP-001
 servers:
-- url: 'https://admin-shell.io/api/v3.1'
-- url: 'https://example.com/'
-
+- url: '{protocol}://{host_name}:{port}/api/{version_prefix}'
```

Comment:
Didn't we decide to support very flexible URL, just {protocol}://{base-url}. + Text: We recommend to use major version of API version only. After version it is still possible that there is a tenant-ID or a different order...

Reply:
This is 3.0